docs/resources/Emoji.md

```diff
@@ -0,0 +1,60 @@
+# Emoji Resource
+
+>warn
+> Routes for controlling emojis do not follow the normal rate limit conventions. These routes are specifically limited on a per-guild basis to prevent abuse. This means that the quota returned by our APIs may be inaccurate, and you may encounter 429s.
```

Comment:
please add a note to the rate limits page too

Reply:
Could we like...fix this? Lol

Reply:
we are going to be changing how rate limits work in v7, b1nzy will be spearheading the changes, or so he says